code/game/objects/items/eightball.dm

```diff
@@ -41,6 +41,13 @@
 		new /obj/item/toy/eightball/haunted(get_turf(src))
 		qdel(src)
 
+/obj/item/toy/eightball/examine(mob/user)
+	..()
+	if(on_cooldown)
```

Comment:
You don't need the time left, just saying that you can't use is enough(and it already does that).